app/controllers/application_controller.rb

```diff
@@ -47,6 +47,10 @@ def decorated_session
     @_decorated_session ||= DecoratedSession.new(sp: current_sp, view_context: view_context).call
   end
 
+  def signed_in_path
```

Comment:
I'm fine with removing the application controller spec. Since this method is called in pretty much every feature spec, I think it's got good coverage. Is that what you're proposing?